src/lib/zod/device.ts

```diff
@@ -66,6 +66,7 @@ export const lock_device_properties = common_device_properties.extend({
 })
 
 export const thermostat_device_properties = common_device_properties.extend({
+  manufacturer: z.string().optional(),
```

Comment:
This should be a common device property, see here: 

https://github.com/seamapi/seam-connect/blob/main/public-types/unstable/models/managed-device.ts#L28

Reply:
updated in https://github.com/seamapi/fake-seam-connect/pull/101/commits/cd2e39d2e43a7cfdd787f63f03b3584e5e1748a4